Makefile

```
@@ -59,7 +59,7 @@ BUILD_DATE=$(shell date +"%Y%m%d-%T")
 # source: https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables
 ifndef GITHUB_ACTIONS
 	BUILD_USER?=$(USER)
-	BUILD_BRANCH?=$(shell git branch --show-current)
+	BUILD_BRANCH?=$(shell git rev-parse --abbrev-ref HEAD)
```

Thanks for the details! IIUC this should work and fallback to your suggestion in case Git doesn't support the newer option:
```suggestion
	BUILD_BRANCH?=$(shell git branch --show-current || git rev-parse --abbrev-ref HEAD)
```

---

I don't really have a good way to test this change in my environment.  I'll see if I can come up with something but if you have tested it yourself then I don't see why this couldn't work.  My only concern was the "else" side of the `||` might need a `shell` too but I'm not sure.

---

I tested and it worked in my env.